### ISSUE TITLE
3.5-dev: GQL error when trying to alias user photo

### ISSUE THREAD

**Issue by FreekVR** [craftcms/cms#6418]
### Description
After updating from 3.5-beta3 to the latest commits I am getting internal server errors when aliasing the user photo field to something else using GQL. Unfortunately the error doesn't seem to be logged anywhere either.

Aliasing any asset fields for an entry works fine. Aliasing other fields beyond the title for users also yields no errors.

### Steps to reproduce

1. The following GQL query:
```
{
  users {
    somethingElse: photo {
      url
    }
  }
}
```
2.  Yields this result:
```
{
  "errors": [
    {
      "message": "Internal server error",
      "category": "internal",
      "locations": [
        {
          "line": 3,
          "column": 5
        }
      ],
      "path": [
        "users",
        0,
        "somethingElse"
      ]
    },
```

Edit: I was able to retrieve a debug message on a local environment:

```
"debugMessage": "Argument 1 passed to craft\\gql\\base\\InterfaceType::resolveElementTypeName() must be an instance of craft\\base\\Element, array given, called in /var/www/html/craft/vendor/webonyx/graphql-php/src/Type/Definition/InterfaceType.php on line 101",
``` 

### Additional info

- Craft version: 3.5.x-dev
- PHP version: 7.2

**Comment by brandonkelly** [comment]
Thanks for pointing that out! This is happening because `photo` and a few other fields have special cases in the eager loading logic (e.g. entries’ `author` fields), so we need to identify all of those and avoid using eager-loading aliases in those cases, from the GraphQL side.

**Comment by andris-sevcenko** [comment]
Okay. This is fixed for 3.5. Also cleaned up quite a bit of eager-loading configuration logic for GraphQL queries.

**Comment by brandonkelly** [comment]
Craft 3.5 RC2 is out now with that fix.
